R/sf.R

```diff
@@ -438,23 +562,85 @@ sf_rescale01_x <- function(x, range) {
 }
 
 
-#' @param crs Use this to select a specific CRS. If not specified, will
-#'   use the CRS defined in the first layer.
+#' @param crs Use this to select a specific coordinate reference system (CRS).
+#'   If not specified, will use the CRS defined in the first layer.
 #' @param datum CRS that provides datum to use when generating graticules
+#' @param label_axes Character vector or named list of character values
+#'   specifying which graticules (meridians or parallels) should be labeled on
+#'   which side of the plot. Meridians are indicated by `"E"` (for East) and
+#'   parallels by `"N"` (for North). Default is `"--EN"`, which specifies
+#'   (clockwise from the top) no labels on the top, none on the right, meridians
+#'   on the bottom, and paralleles on the left. Alternatively, this setting could have been
+#'   specified with `list(left = "N", bottom = "E")`.
+#'
+#'   This parameter can be used alone or in combination with `label_graticules`.
+#' @param label_graticules Character vector indicating which graticules should be labeled
+#'   where. Meridians run north-south, and the letters `"N"` and `"S"` indicate that
+#'   they should be labeled on their north or south end points, respectively.
+#'   Parallels run east-west, and the letters `"E"` and `"W"` indicate that they
+#'   should be labeled on their east or west end points, respectively. Thus,
+#'   `label_graticules = "SW"` would label meridians at their south end and parallels at
+#'   their west end, whereas `label_graticules = "EW"` would label parallels at both
+#'   ends and meridians not at all. Because meridians and parallels can in general
+#'   intersect with any side of the plot panel, for any choice of `label_graticules` labels
+#'   are not guaranteed to reside on only one particular side of the plot panel.
```

Comment:
Include an example? Or maybe it's time to start a sf vignette? (Which would be a separate issue)

Reply:
I think this is a separate issue. Currently, all the sf documentation is combined into one .Rd file, and I think we're accumulating too much material to document everything carefully in this way. My proposal would be to start breaking things up, e.g. make a separate .Rd file just for `coord_sf()`. There it would make sense to give more specific examples of how `coord_sf()` works.

(As an aside, I would also like to consider breaking up the file `sf.R` into `coord-sf.R`, `geom-sf.R`, `stat-sf.R`, etc.)